google/cloud/pubsub/publisher.cc

```diff
@@ -19,8 +19,9 @@ namespace cloud {
 namespace pubsub {
 inline namespace GOOGLE_CLOUD_CPP_PUBSUB_NS {
 
-Publisher::Publisher(std::shared_ptr<PublisherConnection> connection,
-                     PublisherOptions)
+Publisher::Publisher(
+    std::shared_ptr<PublisherConnection> connection,
+    PublisherOptions)  // NOLINT(performance-unnecessary-value-param)
```

Comment:
Q: do you know why this is needed now?

Reply:
A: Honestly, no... `clang-tidy` yelled at me, so I changed it. (It also yelled at me to make the change in `flow_controlled_publisher_connection_test.cc`)